typescript/packages/lookslike-high-level/src/recipes/smolIframe.tsx

```diff
@@ -2,66 +2,43 @@ import { h } from "@commontools/html";
 import { recipe, UI, NAME } from "@commontools/builder";
```

Comment:
soft request on making this a different recipe, having a barebones iframe example is useful for manual testing

Reply:
@jsantell will do before merge